### PR TITLE
DOMA-1567 Fix duplicate ticket comments

### PR DESCRIPTION
**Pull request by dkoviazin**
1. There is no easy way to add DISTINCT to ARRAY_AGG with ORDER BY
2. So, we add id to SELECT (as content can be duplicated in different comments) and then make unique list on creating excel

### CODE REVIEW

File: apps/condo/domains/ticket/schema/ExportTicketsService.js [hunk 1]
```diff
@@ -63,7 +64,7 @@ const ExportTicketsService = new GQLCustomSchema('ExportTicketsService', {
                         operator: ticket.operator || ticket.createdBy || '',
                         executor: ticket.executor || '',
                         assignee: ticket.assignee || '',
-                        comments: ticket.TicketComment.join('\n'),
+                        comments: comments.map(comment => comment.split(':').pop()).join('\n' + '—'.repeat(20) + '\n'),
```
Comment (AntonAL) [inline apps/condo/domains/ticket/schema/ExportTicketsService.js:67]: Please, explain in comment, what this 20 dashes are used for.

File: apps/condo/domains/ticket/utils/serverSchema/index.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ const loadTicketsForExcelExport = async ({ where = {}, sortBy = ['createdAt_DESC
         ],
         multipleRelations: [
             [
-                (idx, knex) => knex.raw(`ARRAY_AGG(mr${idx}.content ORDER BY mr${idx}."createdAt" ASC) as "TicketComment"`),
+                (idx, knex) => knex.raw(`ARRAY_AGG(mr${idx}.id || ':' || mr${idx}.content ORDER BY mr${idx}."createdAt" ASC) as "TicketComment"`),
```
[the remark this answers is not in the document]
Comment (dkoviazin) [reply apps/condo/domains/ticket/utils/serverSchema/index.js:67]: There is no user data here 
1. Step - use graphql to fetch ids of exported tickets by user request (standard graphql syntax)
2. Step to build query for loading all relations in one SQL request based on the list of ids

File: apps/condo/domains/ticket/schema/ExportTicketsService.js (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/domains/ticket/schema/ExportTicketsService.js:67]: It's a separator for comments - will make comment

Comment (dkoviazin) [reply apps/condo/domains/ticket/schema/ExportTicketsService.js]: Fixed

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/ticket/schema/ExportTicketsService.js [hunk 3]
```diff
@@ -63,7 +66,7 @@ const ExportTicketsService = new GQLCustomSchema('ExportTicketsService', {
                         operator: ticket.operator || ticket.createdBy || '',
                         executor: ticket.executor || '',
                         assignee: ticket.assignee || '',
-                        comments: ticket.TicketComment.join('\n'),
+                        comments: comments.map(comment => comment.split(':').pop()).join(TICKET_COMMENTS_SEPARATOR),
```
Comment (akarjakin) [inline apps/condo/domains/ticket/schema/ExportTicketsService.js:69]: as far as I know pop returns either last element of an array or undefined, so `.join(...)` here will do nothing...

**Review by akarjakin — APPROVED** [review]

**Pull request merged**